src/middlewared/middlewared/plugins/account.py

```diff
@@ -292,7 +292,7 @@ async def query(self, filters, options):
         if dssearch:
```

Comment:
While not part of your changes, we could clean this module up by changing the import on line 18 from:
`from middlewared.api.current import *`
to:
```
from middlewared.api.current import (
    UserEntry, UserCreateArgs, UserCreateResult, UserUpdateArgs, UserUpdateResult
)
```